ui/raidboss/data/04-sb/raid/o11s.js

```diff
@@ -498,24 +504,25 @@
         'Repel': '相斥',
         'Reset': '初始化',
         'Rush': '突进',
-        'Starboard Wave Cannon Surge': '右舷齐射·零式波动炮',
         'Starboard Wave Cannon': '右舷齐射·波动炮',
+        'Starboard Wave Cannon Surge': '右舷齐射·零式波动炮',
```

Comment:
Similar questions here about `Starboard Wave Cannon` conflicting with `Starboard Wave Cannon Surge`.  I think your reordering breaks this.

Right now, all of these apply in order, so having Surge before non-Surge works properly.  However, alphabetizing breaks this.  I am not opposed to the idea that exactly one sync replacement should apply to a text, but it would be better to have testing for that and to add extra regex like ^ and $ where appropriate to ensure that.

Reply:
done with lookahead/lookbehind